src/view/editableelement.js

```diff
@@ -105,48 +106,6 @@ export default class EditableElement extends ContainerElement {
 	destroy() {
 		this.stopListening();
 	}
-
-	/**
-	 * Returns document associated with the editable.
-	 *
-	 * @readonly
-	 * @returns {module:engine/view/document~Document}
-	 */
-	get document() {
-		return this.getCustomProperty( documentSymbol );
-	}
-
-	/**
-	 * Sets document of this editable element.
-	 *
-	 * @protected
-	 * @param {module:engine/view/document~Document} document
-	 */
-	set _document( document ) {
-		if ( this.getCustomProperty( documentSymbol ) ) {
-			/**
-			 * View document is already set. It can only be set once.
-			 *
-			 * @error view-editableelement-document-already-set
-			 */
-			throw new CKEditorError( 'view-editableelement-document-already-set: View document is already set.', this );
```

Comment:
@Reinmar do you remember why this was throwing error?

Reply:
NO. Probably for the reason of sanity. Roots should not be moved around